### DISCUSSION TITLE
t_smtp.localip

### DISCUSSION THREAD

**Discussion by lingtiao** [Jinnrry/PMail#232] [General]
邮箱会不定时收到这样的邮件，请问是怎么回事，我该如何解决它
![1](https://github.com/user-attachments/assets/588ef13a-1621-40a1-9ad6-fad459b78672)

**Comment by Jinnrry** [comment]
垃圾邮件，探测你服务器是否有smtp服务。就类似你在公网开一个22端口，立马会有无数多脚本来尝试爆破你的密码。

https://github.com/Jinnrry/PMail/tree/master/server/hooks/spam_block

可以尝试垃圾邮件屏蔽插件，但是这个插件训练得不是很好，你可以自己训练。或者使用收件规则，自己写正则屏蔽也行

**Comment by lingtiao** [comment]
好的，感谢大大发自我的 iPhone在 2024年12月30日，11:03，Jinnrry ***@***.***> 写道：﻿
垃圾邮件，探测你服务器是否有smtp服务。就类似你在公网开一个22端口，立马会有无数多脚本来尝试爆破你的密码。
https://github.com/Jinnrry/PMail/tree/master/server/hooks/spam_block
可以尝试垃圾邮件屏蔽插件，但是这个插件训练得不是很好，你可以自己训练。或者使用收件规则，自己写正则屏蔽也行

**Reply by AXFOX** [reply]
家里NAS有核显支持Vulkan的话可以试试我的项目：https://github.com/AXFOX/Tranquil-Inbox-Ward 
git clone https://github.com/AXFOX/Tranquil-Inbox-Ward
cd Tranquil-Inbox-Ward && sudo bash ./deploy-systemd.sh
在Pmail里Spam插件添加URL http://<NAS_IP:8501>/v1/models/emotion_model:predict 即可

**Comment by lingtiao** [comment]
good发自我的 iPhone在 2026年2月15日，07:49，xfox ***@***.***> 写道：﻿
家里NAS有核显支持Vulkan的话可以试试我的项目：https://github.com/AXFOX/Tranquil-Inbox-Ward
